### PR TITLE
Bump Gluon to v2023.2 and remove x86-64 workaround

### PR DESCRIPTION
**Pull request by grische**

### CODE REVIEW

**Comment by grische** [comment]
It seems we do not need the x86-64 build workaround anymore 😱

File: image-customization.lua [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+}
+
+include_tls = not device({
+    'd-link-dir825b1',
+})
+
+if include_tls then
+    features({ 'tls' })
+    packages({
+        'ca-bundle',
```
Comment (maurerle) [inline image-customization.lua]: included in feature-tls

File: image-customization.lua [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    'kmod-usb-net-smsc95xx',
+}
+
+pkgs_pci = {
+    'pciutils',
+    'kmod-bnx2', -- Broadcom NetExtreme BCM5706/5708/5709/5716
+}
+
+include_tls = not device({
+    'd-link-dir825b1',
```
Comment (maurerle) [inline image-customization.lua:110]: The dir825b1 is also part of `tiny` - so we could move TLS to if not tiny?

File: image-customization.lua [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    'web-private-wifi',
+    'web-wizard',
+}
+
+packages{
+    'ffac-autoupdater-wifi-fallback',
+    'ffac-change-autoupdater',
+    'ffac-ssid-changer',
+    'ffho-ap-timer',
+    'ffho-web-ap-timer',
```
Comment (maurerle) [inline image-customization.lua]: switch to community ap-timer?

File: Makefile [hunk 4]
```diff
@@ -1,6 +1,6 @@
 GLUON_BUILD_DIR := gluon-build
 GLUON_GIT_URL := https://github.com/freifunk-gluon/gluon.git
-GLUON_GIT_REF := dc841ea4a37b4e74777e07617042bc92aad2483b # master at 2023-12-13
+GLUON_GIT_REF := 06d547710bcd1d0e99ed6acb54002f9a46ed23de
```
Comment (maurerle) [inline Makefile]: v2023.2?

File: image-customization.lua (hunk 2 above)
Comment (grische) [reply image-customization.lua:110]: @maurerle from what I can tell, all other devices, except dir825b1 build fine with TLS even if they are tiny.

**Review by GoliathLabs — APPROVED** [review]

**Pull request merged**